libs/ledger-live-common/src/families/xrp/setup.ts

```diff
@@ -0,0 +1,31 @@
+// Goal of this file is to inject all necessary device/signer dependency to coin-modules
+
+import Xrp from "@ledgerhq/hw-app-xrp";
+import { Bridge } from "@ledgerhq/types-live";
+import Transport from "@ledgerhq/hw-transport";
+import { XrpConfig } from "@ledgerhq/coin-xrp/lib/config";
+import xrpResolver from "@ledgerhq/coin-xrp/hw-getAddress";
+import { createBridges } from "@ledgerhq/coin-xrp/bridge/js";
+import makeCliTools from "@ledgerhq/coin-xrp/cli-transaction";
+import { Transaction as XrpTransaction } from "@ledgerhq/coin-xrp/types";
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
+import { CreateSigner, createResolver, executeWithSigner } from "../../bridge/setup";
+import { getCurrencyConfiguration } from "../../config";
+import { Resolver } from "../../hw/getAddress/types";
+
+const createSigner: CreateSigner<Xrp> = (transport: Transport) => {
+  return new Xrp(transport);
+};
+
+const getCoinConfig = () => getCurrencyConfiguration<XrpConfig>(getCryptoCurrencyById("ripple"));
```

Comment:
Shouldn't be "xrp" instead?

Reply:
Unfortunately I can't change the currency name without removing every XRP account from all Ledger Live instances... That wouldn't problematic to add them again, but I guess that's a big impact on CS for a small change like this 🤷‍♂️